src/components/Keyword/KeywordList/index.js

```diff
@@ -0,0 +1,329 @@
+import React,{useCallback,useEffect,useState,useRef} from 'react';
+import KeywordHeader from '../KeywordHeader';
+import * as S from './styles';
+import { AUNURI,AOUMIR } from 'constant';
+import { useSelector, useDispatch } from 'react-redux';
+import { getKeywordList,deleteKeywordList,moveKeywordItem,readKeywordItem } from 'store/keyword';
+import { getTitle,makeDeleteQuery } from '../utills';
```

Comment:
누가 만들었는지 모르겠지만 파일 이름 수정하는게 좋아보입니다

```suggestion
import { useSelector, useDispatch } from 'react-redux';
import { getKeywordList,deleteKeywordList,moveKeywordItem,readKeywordItem } from 'store/keyword';
import { getTitle,makeDeleteQuery } from '../utils';
```

Reply:
제가 만든 파일입니다! 수정하도록 하겠습니다

Reply:
86aaf19

Reply:
😅